app/views/posts/index.html.erb

```diff
@@ -1,41 +1,32 @@
 <p id="notice"><%= notice %></p>
 
-<h1>Posts</h1>
+<div class="container d-flex">
+  <div class="row align-self-center"><h2>Posts</h2></div>
+</div>
 
-<table>
-  <thead>
-    <tr>
-      <th>Title</th>
-      <th>Body</th>
-      <th>Author</th>
-      <th>Replies count</th>
-      <th>Votes count</th>
-      <th>Favorites count</th>
-      <th>Follows count</th>
-      <th>Tags count</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
+<div class="container">
+  <% @posts.each do |post| %>
+      <div class="d-flex border-top">
+        <div class="pt-4 h5 text-info"><%= post.title %></div>
+      </div>
+      <div class="pb-2 d-flex">
+        <div><%= post.body %></div>
+      </div>
 
-  <tbody>
-    <% @posts.each do |post| %>
-      <tr>
-        <td><%= post.title %></td>
-        <td><%= post.body %></td>
-        <td><%= post.author_id %></td>
-        <td><%= post.replies_count %></td>
-        <td><%= post.votes_count %></td>
-        <td><%= post.favorites_count %></td>
-        <td><%= post.follows_count %></td>
-        <td><%= post.tags_count %></td>
-        <td><%= link_to 'Show', post %></td>
-        <td><%= link_to 'Edit', edit_post_path(post) %></td>
-        <td><%= link_to 'Destroy', post, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
+        <div class="d-flex flex-row-reverse">
+          <div><%= image_tag post.author.avatar_img, style: "height:30px"%></div>
```

Comment:
It's usually not recommended to use inline styles unless it's on icon maybe. It might be better to extract this to a custom CSS class ([if bootstrap doesn't have what you need](https://getbootstrap.com/docs/4.6/content/images/#image-thumbnails)) that you can use later in your project if you need it.

Reply:
To be clear, is it both divs or just the one with the image_tag that concerns you here?

Reply:
@cole-john Ah, it's actually just the `image_tag` on line 17 that has the inline style.

Reply:
Got it. I understand and will make the adjustments via custom css. Thanks.